ip-bom/pom.xml

```diff
@@ -2972,18 +2898,6 @@
       </dependency>
 
       <dependency>
-        <groupId>org.xmappr</groupId>
-        <artifactId>xmappr</artifactId>
-        <version>${version.org.xmappr}</version>
```

Comment:
Good point, it should! (Assuming it is not used in some other place). This maybe case for some other properties as well, will take a look.